packages/swingset-runner/bin/graphDisk

```diff
@@ -0,0 +1 @@
+../src/graphDisk.js
```

Comment:
I'm still a bit worried about putting symlinks into git, but @FUDCo pointed out:

* these are just convenience aliases (it's easier to discover that something is a top-level runnable script when it's in `bin/`), and you can always run `node -r esm src/graphDisk.js` instead
* windows has "file aliases" which are like symlinks, so `git` might conceivably use them during `git clone`

So we decided to run with it, but to keep an ear open for any problem reports from windows users.